### DISCUSSION TITLE
[A-L] (2023/24) Foglio 1 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#134] [Esercizi - Bertini]
![Es 3](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/b877307d-edd3-44dc-800e-0b05cb063a34)

**Comment by Elia-Belli** [comment]
$|A \cup B \cup C| = |A| + |B| + |C| - |A \cap B| - |B \cap C| - |A \cap C| + |A \cap B \cap C|$

**Comment by CiottoloMaggico** [comment]
Analogo a @Elia-Belli

**Comment by FeddyLix17** [comment]
soluzione di @Elia-Belli corretta, propongo una dimostrazione del principio di [Inclusione-Esclusione](https://www.youtube.com/watch?v=qkMccoTKLBE) di seguito

rappresentando i 3 insiemi usando il [diagramma di Eulero-Venn](https://www.youmath.it/lezioni/algebra-elementare/insiemistica/566-diagramma-di-eulero-venn.html)

<div align="center">

![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/66442720/ccea2dc3-daa7-41d2-9e52-994bdfe169e8)

</div>

- conteggiando una volta tutti gli elementi di $\huge A$, conteggeremo una prima volta gli elementi di $\huge A \cap B$, quelli di $\huge A \cap C$ e infine gli elementi di $\huge A \cap B \cap C$

- conteggiando successivamente una volta tutti gli elementi di $\huge B$,  verranno conteggiati una prima volta gli elementi di $\huge B \cap C$, ma risulteranno conteggiati una seconda volta gli elementi di $\huge A \cap B$ e quelli di $\huge A \cap B \cap C$

- conteggiando infine una volta tutti gli elementi di $\huge C$, conteggeremo una seconda volta gli elementi di $\huge B \cap C$, e $\huge A \cap C$, mentre gli elementi di $\huge A \cap B \cap C$ andranno ad essere conteggiati una terza volta

si arriva dunque ad una situazione nella quale vengono conteggiati 2 volte $\huge A \cap B,\ A \cap C,\ B \cap C$ e per tre volte $\huge A \cap B \cap C$.

- rimuovendo una volta $\huge |A \cap B|$ dal conteggio, verranno anche tolti una volta gli elementi $\huge |A \cap B \cap C|$
- rimuovendo una volta $\huge |A \cap C|$, gli elementi $\huge |A \cap B \cap C|$ saranno rimossi una seconda volta
- e togliendo infine una volta gli elementi $\huge |B \cap C|$, gli elementi $\huge |A \cap B \cap C|$ saranno rimossi una terza (non voluta) volta

Per avere l'esatto numero di elementi di $\huge A \cup B \cup C$ bisognerà riaggiungere una volta gli elementi $\huge |A \cap B \cap C|$, ricollegandosi cosi alla [soluzione](https://github.com/sapienzastudentsnetwork/calcprob2324/issues/03#issuecomment-1745483429) di @Elia-Belli riportata sopra
